pixivpy3/aapi.py

```diff
@@ -65,6 +65,10 @@ def __init__(self, **requests_kwargs: Any) -> None:
         """initialize requests kwargs if need be"""
         super(AppPixivAPI, self).__init__(**requests_kwargs)
 
+    # add cloudscraper proxies
+    def set_cloudscraper_proxy(self,proxy_local: dict = {'http': 'http://127.0.0.1:7890', 'https': 'https://127.0.0.1:7890'}):
```

Comment:
I wouldn't use dict (i. e. mutual object) as a default argument within the signature of the method.

Mutual default values for arguments can cause unpredictable behavoiur.
```
>>> def foobar(bar=[]):
...     bar.append('test')
...     print(f'List {bar} with ID {id(bar)}')
>>>
>>>
>>> foobar()
List ['test'] with ID 140147921265664
>>> foobar()
List ['test', 'test'] with ID 140147921265664  # as you see function hasn't create an empty list during second call, instead added value to the existing one
```
Instead i would suggest you to use pattern like this:
```
...
def set_cloudscraper_proxy(self, proxy_local: Optional[dict]):
    if proxy_local is None:
        proxy_local = {
            "http": "http://127.0.0.1:7890", "https": "https://127.0.0.1:7890"
        }
        self.requests.proxies = proxy_local
...
```

___

Another point, are you sure that default values for proxies should be on the localhost and the same port using for `http` as well as for `https` (7890)? It looks quite strange for me, well, i haven't seen 7890 port before as a default for something. I'd be glad if you told me why you had added exactly that number.

___

And the last minor note, i guess linter would fail when validating this commit. Considering there is no whitespace between arguments (at the method's definition: `self,proxy_local:` should be `self, proxy_local:`). Also replace single quotes (') with double ("). You can make this easily via black in a several comments:
```
$ pip install black
$ black pixivpy3/aapi.py
```

Reply:
Thanks for your kind suggestions. I ran into issues with 2captcha and incorrect proxies from my Clash app when using the pixivpy API a few days ago. I believe the problems arose from the GFW and incorrect proxies. To bypass the verification in the auth.py API, I tried adding proxies to the cloudscraper. However, I wrongly committed some changes without properly testing and modifying the code. Please close that pull request. I will commit again after further testing and modifying the code.